lib/tasks/multitenant/tenants.rake

```diff
@@ -68,6 +68,37 @@ namespace :multitenant do
       update_tenant_ids(proc { |object| object.account.tenant_id }, proc { account }, proc { tenant_id == nil }, args.to_hash.merge({ table_name: 'Alert' }))
     end
 
+    desc 'Check and remove orphaned objects (whose tenant is missing), pass "destroy" argument to delete'
+    task :cleanup_orphans, [:mode] => :environment do |_task, args|
+      puts 'Checking and removing orphaned objects...'
+
+      destroy = args[:mode] == "destroy"
```

Comment:
So, the idea is that by default calling the task would only print the logs.
To actually destroy stuff, one would need to run 
```
bundle exec rake "multitenant:tenants:cleanup_orphans[destroy]"
```